app/src/main/AndroidManifest.xml

```diff
@@ -34,7 +35,7 @@
         <activity
             android:name=".MainActivity"
             android:label="@string/title_activity_maps"
-            android:screenOrientation="portrait">
+            android:screenOrientation="fullSensor">
```

Comment:
This alteration is causing the app to crash.
For example: 
- you click on the toggle button to display SGW campus
- while the map is loading the set coordinates, you change your orientation to landscape
- app crash

We should revert this back to *portrait*.

Reply:
Woops! Good catch! I did not notice it. I will remove it right away 😄